src/index.js

```diff
@@ -47,6 +54,17 @@ export function useQuery(query, { suspend = true, ...restOptions } = {}) {
     refetch: variables => observableQuery.current.refetch(variables),
     startPolling: interval => observableQuery.current.startPolling(interval),
     stopPolling: () => observableQuery.current.stopPolling(),
+    subscribeToMore: opts => {
+      const { document, variables } = opts;
+      if (
+        document !== previousSubscription.current ||
+        isEqual(variables, previousSubscribtionOptions.current)
+      ) {
+        unsubscribeToMore.current = observableQuery.current.subscribeToMore(
```

Comment:
`subscribeToMore` returns a function to unsubscribe the subscription [see here](https://github.com/apollographql/apollo-client/blob/911ca2db631adc38745b8098dd802e0cdfe8dbbb/packages/apollo-client/src/core/ObservableQuery.ts#L390)

Reply:
conditional mistake corrected in the commit below